app/test_main.py

```diff
@@ -1,3 +1,27 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+class TestCatDogYears:
+    @pytest.mark.parametrize(
+        "cat_years, dog_years, human_years",
+        [
+            pytest.param(
+                15, 15,
+                [1, 1],
+                id="should return 1 human year for first animal years"
+            ),
+            pytest.param(
```

Comment:
Could you add a test [23, 23] to make sure that the function uses exactly a 9-year difference?